packages/SwingSet/jsconfig.json

```diff
@@ -1,13 +1,8 @@
 // This file can contain .js-specific Typescript compiler config.
 {
+  "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "target": "esnext",
-    "module": "esnext",
-    "checkJs": true,
-    "noEmit": true,
-    "strictNullChecks": true,
-    "noImplicitThis": true,
-    "moduleResolution": "node",
+    "downlevelIteration": false,
   },
```

Comment:
I left options like this to preserve behavior, but I suspect that the `downlevelIteration` in particular doesn't actually need any overrides.

Reply:
let's remove. I'm not sure why it was ever there because `downlevelIteration` is for JS output and we don't transform our JS before publish.